lib/bitcoin/BitcoinProcessor.ts

```diff
@@ -793,7 +800,10 @@ export default class BitcoinProcessor {
       const oneSidetreeTxAlreadyFound = (sidetreeTxToAdd !== undefined);
 
       if (isSidetreeTx && oneSidetreeTxAlreadyFound) {
-        throw new Error('The transaction has more then one sidetree anchor strings.');
+        // tslint:disable-next-line: max-line-length
+        const message = `The outputs in block: ${transactionBlock} with transaction id: ${transactionId} has multiple sidetree transactions. So ignoring this transaction.`;
+        console.debug(message);
+        return false;
```

Comment:
This change looks good, but it raises a question regarding the try-catch block that wraps this method call: should we eat the exception or throw?

Reply:
We shouldn't catch. The catch is there right now only to log debug info.